# Ukrainian translation for ubuntuone-client
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the ubuntuone-client package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
# Sergiy Gavrylov <sergiovana@bigmir.net>, 2010.
msgid ""
msgstr ""
"Project-Id-Version: ubuntuone-client\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2012-09-10 21:26+0000\n"
"PO-Revision-Date: 2011-11-02 21:28+0000\n"
"Last-Translator: sunsays <Unknown>\n"
"Language-Team: Ukrainian <uk@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=n%10==1 && n%100!=11 ? 0 : n%10>=2 && "
"n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2;\n"
"X-Launchpad-Export-Date: 2012-09-17 18:32+0000\n"
"X-Generator: Launchpad (build 15966)\n"

#: ../ubuntuone/status/aggregator.py:51
msgid "Magicicada"
msgstr "Magicicada"

#: ../ubuntuone/status/aggregator.py:52
msgid "New cloud folder(s) available"
msgstr "В хмарі доступна нова тека(и)"

#: ../ubuntuone/status/aggregator.py:53
msgid "File synchronization completed."
msgstr "Синхронізування файлів виконано."

#: ../ubuntuone/status/aggregator.py:55
#, python-format
msgid "%(percentage_completed)d%% completed."
msgstr "%(percentage_completed)d%% виконано."

#: ../ubuntuone/status/aggregator.py:56
msgid "File synchronization in progress"
msgstr "Триває синхронізування файлів."

#: ../ubuntuone/status/aggregator.py:59
#, python-format
msgid ""
"There is no available space on the folder:\n"
"\"%s\" shared by %s"
msgstr ""
"Нема доступного місця у каталозі:\n"
"\"%s\" (у спільному доступі %s)"

#: ../ubuntuone/status/aggregator.py:73
#, python-format
msgid "'%(filename)s' is being uploaded to your personal cloud."
msgstr "'%(filename)s' завантажується у вашу власну хмару."

#: ../ubuntuone/status/aggregator.py:79
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file are being uploaded to your "
"personal cloud."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files are being uploaded to your "
"personal cloud."
msgstr[0] ""
"'%(filename)s' та ще %(other_files)d файл завантажуються до вашої особистої "
"хмари."
msgstr[1] ""
"'%(filename)s' та ще %(other_files)d файли завантажуються до вашої особистої "
"хмари."
msgstr[2] ""
"'%(filename)s' та ще %(other_files)d файлів завантажуються до вашої "
"особистої хмари."

#: ../ubuntuone/status/aggregator.py:90
#, python-format
msgid "'%(filename)s' is being downloaded to your computer."
msgstr "'%(filename)s' завантажується на ваш комп’ютер."

#: ../ubuntuone/status/aggregator.py:96
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file are being downloaded to your "
"computer."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files are being downloaded to your "
"computer."
msgstr[0] ""
"'%(filename)s' та ще %(other_files)d файл завантажуються до вашого "
"комп’ютера."
msgstr[1] ""
"'%(filename)s' та ще %(other_files)d файли завантажуються до вашого "
"комп’ютера."
msgstr[2] ""
"'%(filename)s' та ще %(other_files)d файлів завантажуються до вашого "
"комп’ютера."

#: ../ubuntuone/status/aggregator.py:107
#, python-format
msgid "'%(filename)s' was uploaded to your personal cloud."
msgstr "'%(filename)s' було завантажено у вашу персональну хмару."

#: ../ubuntuone/status/aggregator.py:113
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file were uploaded to your personal "
"cloud."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files were uploaded to your "
"personal cloud."
msgstr[0] ""
"'%(filename)s' та ще %(other_files)d файл завантажено до вашої особистої "
"хмари."
msgstr[1] ""
"'%(filename)s' та ще %(other_files)d файли завантажено до вашої особистої "
"хмари."
msgstr[2] ""
"'%(filename)s' та ще %(other_files)d файлів завантажено до вашої особистої "
"хмари."

#: ../ubuntuone/status/aggregator.py:124
#, python-format
msgid "'%(filename)s' was downloaded to your computer."
msgstr "'%(filename)s' було завантажено на ваш комп’ютер."

#: ../ubuntuone/status/aggregator.py:130
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file were downloaded to your "
"computer."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files were downloaded to your "
"computer."
msgstr[0] ""
"'%(filename)s' та ще %(other_files)d файл завантажено до вашого комп’ютера."
msgstr[1] ""
"'%(filename)s' та ще %(other_files)d файли завантажено до вашого комп’ютера."
msgstr[2] ""
"'%(filename)s' та ще %(other_files)d файлів завантажено до вашого комп’ютера."

#: ../ubuntuone/status/aggregator.py:191
#, python-format
msgid "A file was just made public at %(new_public_url)s"
msgstr "Файл було щойно опубілковано за адресою %(new_public_url)s"

#: ../ubuntuone/status/aggregator.py:204
#, python-format
msgid "%(event_count)d file was just made public."
msgid_plural "%(event_count)d files were just made public."
msgstr[0] "%(event_count)d файл щойно було опубліковано."
msgstr[1] "%(event_count)d файли щойно було опубліковано."
msgstr[2] "%(event_count)d файлів щойно було опубліковано."

#: ../ubuntuone/status/aggregator.py:212
msgid "A file is no longer published"
msgstr "Файл більше не опубліковано"

#: ../ubuntuone/status/aggregator.py:220
#, python-format
msgid "%(event_count)d file is no longer published."
msgid_plural "%(event_count)d files are no longer published."
msgstr[0] "%(event_count)d файл більше не опубліковано."
msgstr[1] "%(event_count)d файли більше не опубліковано."
msgstr[2] "%(event_count)d файлів більше не опубліковано."

#: ../ubuntuone/status/aggregator.py:235
#, python-format
msgid "Found %(event_count)d new cloud folder."
msgid_plural "Found %(event_count)d new cloud folders."
msgstr[0] "Знайдена %(event_count)d нова хмаринкова тека."
msgstr[1] "Знайдено %(event_count)d нові хмаринкові теки."
msgstr[2] "Знайдено %(event_count)d нових хмаринкових тек."

#: ../ubuntuone/status/aggregator.py:243
#, python-format
msgid ""
"New cloud folder available: '%(folder_name)s' shared by %(other_user_name)s"
msgstr ""
"Доступна нова хмаринкова тека: '%(folder_name)s' опублікована "
"%(other_user_name)s"

#: ../ubuntuone/status/aggregator.py:259
#, python-format
msgid "New cloud folder available: '%(folder_name)s'"
msgstr "Доступна нова хмаринкова тека: '%(folder_name)s'"

#: ../ubuntuone/status/aggregator.py:281
msgid "The connection to the server was lost."
msgstr "З’єднання з сервером було втрачено."

#: ../ubuntuone/status/aggregator.py:287
msgid "The connection to the server was restored."
msgstr "З’єднання з сервером відновлено."

#: ../data/emblem-ubuntuone-downloading.icon.in.h:1
msgid "Downloading"
msgstr "Завантаження"

#: ../data/emblem-ubuntuone-unsynchronized.icon.in.h:1
msgid "Unsynchronized"
msgstr "Не синхронізовано"

#: ../data/emblem-ubuntuone-uploading.icon.in.h:1
msgid "Uploading"
msgstr "Вивантаження"
